sycl/test-e2e/Graph/RecordReplay/empty_no_recording.cpp

```diff
@@ -0,0 +1,42 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
```

Comment:
So far we've limited out tests to just testing the graph extension rather than core SYCL code, so I'd tend towards removing this test unless is stresses our extension in some way